src/app/views/data/data.js

```diff
@@ -30,6 +54,28 @@ class DataViewController extends ADBMapController {
   _addGeoJSONLayer(geojson) {
     L.geoJson(geojson).addTo(this.map);
   }
+
+  _onResize() {
+    this.charts.width = document.querySelectorAll(".histogram .panel-content p")[0].clientWidth;
+    try {
```

Comment:
I'll move this over to use that, I saw some articles on SO that suggested `$timeout`, not sure why I didn't do that here.